CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Wayland's Client side decorations now use the search bar colors
 - Reduce memory usage by up to at least 30% with a full scrollback buffer
 - The number of zerowidth characters per cell is no longer limited to 5
+- `SpawnNewInstance` is now using working directory of active process on macOS
```

Comment:
```suggestion
- `SpawnNewInstance` is now using the working directory of the terminal foreground process on macOS
```

"Active process" is honestly kind of a poor name for it. I know that the other changelog entry uses this wording already but that's just a bit misleading. The `terminal` is kinda optional, but I think it's a useful addition.

I'm open to changing the other changelog entry too.